examples-go/deploy-model/main.go

```diff
@@ -85,9 +85,10 @@ func main() {
 
 	_, err = c.UpdateModel(ctx, &modelPB.UpdateModelRequest{
 		Model: &modelPB.UpdateModelInfo{
+			Name:   "ensemble",
```

Comment:
Can we change the model name to `yolov4` to be more specific?

Reply:
@xiaofei-du okie let me change and upload again into gcs

Reply:
I think `ensemble` is too general. Any model can be an ensembled model of pre-processing, infer and post-processing model.

Currently, we don't allow calling pre-processing, infer and post-processing models, so naming it as `ensemble` somehow makes me feel that we also allow the user to make inferences with the rest.

Reply:
@xiaofei-du how about `yolov4-ensemble`

Reply:
We can just keep it simple. For users who are not familiar with Triton, `ensemble` is a concept that needs to be explained in documentation.